resources/shaders/deferred_composition.hlsl

```diff
@@ -16,8 +16,8 @@ TextureCube irradiance_map   : register(t5);
 TextureCube pref_env_map	 : register(t6);
 Texture2D brdf_lut			 : register(t7);
 Texture2D buffer_refl_shadow : register(t8); // xyz: reflection, a: shadow factor
-Texture2D screen_space_irradiance : register(t9);
-Texture2D screen_space_ao : register(t10);
+Texture2D gbuffer_AO 		 : register(t9); //x: AO value
```

Comment:
This is not a gbuffer :O. the old name is good enough.

Reply:
I adopted the old name, misunderstood the use of screen_space in the naming, also had not clue what a GBuffer actually was 😨